src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -14,13 +14,18 @@ import ZUIRelativeTime from 'zui/ZUIRelativeTime';
 import {
   DataGridPro,
   GridCellParams,
+  GridComparatorFn,
   GridRenderCellParams,
   useGridApiContext,
 } from '@mui/x-data-grid-pro';
 import { FC, useEffect, useMemo } from 'react';
 import { Msg, useMessages } from 'core/i18n';
 import { ZetkinPerson, ZetkinSurveySubmission } from 'utils/types/zetkin';
 
+const alphabeticalComparator: GridComparatorFn<string> = (v1, v2) =>
+  //compares strings alphabetically, locale aware
+  v1.localeCompare(v2);
```

Comment:
Nice way that sorting with `sortComparator` that gets value from `valueGetter`! I just tested it, and got this error from `alphabeticalComparator` when there is no value in a cell.
![image](https://github.com/zetkin/app.zetkin.org/assets/77925373/38e5d46a-00fb-44f9-b210-759537c22ced)